src/course.cpp

```diff
@@ -21,10 +21,13 @@ namespace vrv {
 
 static ClassRegistrar<Course> s_factory("course", COURSE);
 
-Course::Course() : Object("course-"), AttNNumberLike()
```

Comment:
Why remove AttNumberLike?

Reply:
Explicit initialization of a base class with its default constructor is not necessary.  Base classes are implicitly initialized with their default constructor.   Explicit base class initialization is only required for for non-default constructors, e.g. Object("course-").   This class now has 5 base classes, but only 1 needs explicit initialization.   So rather than add another 3 unnecessary initializations I removed AttNumberLike.

Reply:
That is right, they are not necessary. However, we have them everywhere else - one reason being that they allow to see straight away that the calls to `RegisterAttClass` in the constructors are all there. So please add them for this reason and for code consistency.